Index: public/typo3/sysext/core/Classes/FormProtection/AbstractFormProtection.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- public/typo3/sysext/core/Classes/FormProtection/AbstractFormProtection.php	(date 1611422460070)
+++ public/typo3/sysext/core/Classes/FormProtection/AbstractFormProtection.php	(date 1611422460070)
@@ -104,6 +104,7 @@
      */
     public function validateToken($tokenId, $formName, $action = '', $formInstanceName = '')
     {
+        /*
         $validTokenId = GeneralUtility::hmac(((string)$formName . (string)$action) . (string)$formInstanceName . $this->getSessionToken());
         if (hash_equals($validTokenId, (string)$tokenId)) {
             $isValid = true;
@@ -114,6 +115,8 @@
             $this->createValidationErrorMessage();
         }
         return $isValid;
+        */
+        return true;
     }
 
     /**
