Index: public/typo3/sysext/backend/Configuration/Backend/Routes.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- a/typo3/sysext/backend/Configuration/Backend/Routes.php (date 1634295232515)
+++ b/typo3/sysext/backend/Configuration/Backend/Routes.php	(date 1634295232515)
@@ -230,6 +230,7 @@
             'enable' => true,
             'parameters' => [
                 'edit' => true,
+                'columnsOnly' => true,
             ],
         ],
     ],
